.github/workflows/ci.yaml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - main
+            - '[0-9]+.[0-9]+'
+    pull_request: ~
+
+jobs:
+    cs_fix:
```

Comment:
I see we have some inconsistency in the style of key names. Here it's `_`, in the other cases it's `-`. Don't you think it would be better if he had some common pattern?

Reply:
Sure, changed to `-` in https://github.com/ezsystems/doctrine-dbal-schema/pull/27/commits/c33ad2158a2199733f8009807626bb9c229d7a24